### PR TITLE
CICE initialization from analysis

### PR DESCRIPTION
**Pull request by guillaumevernieres**

### CODE REVIEW

File: jobs/JGDAS_GLOBAL_OCEAN_ANALYSIS_POST [hunk 1]
```diff
@@ -11,9 +11,11 @@ source "${HOMEgfs}/ush/jjob_header.sh" -e "ocnanalpost" -c "base ocnanalpost"
 ##############################################
 
 export COMOUT=${COMOUT:-${ROTDIR}/${CDUMP}.${PDY}/${cyc}/ocean}
+export COMOUTice=${ROTDIR}/${CDUMP}.${PDY}/${cyc}/ice
 export CDATE=${CDATE:-${PDY}${cyc}}
 
-mkdir -p "${COMOUT}"
+mkdir -p "${COMOUT}/RESTART"
+mkdir -p "${COMOUTice}/RESTART"
```
Comment (AndrewEichmann-NOAA) [inline jobs/JGDAS_GLOBAL_OCEAN_ANALYSIS_POST:17]: I'm puzzled about this - was it not making a `RESTART` directory before (for the ocean)?

**Review by AndrewEichmann-NOAA — APPROVED** [review]

File: jobs/JGDAS_GLOBAL_OCEAN_ANALYSIS_POST (hunk 1 above)
Comment (guillaumevernieres) [reply jobs/JGDAS_GLOBAL_OCEAN_ANALYSIS_POST:17]: ... Ha! good catch, I don't need line 17. I need line 18 to save the cice restart that contains the analsyis.

File: jobs/JGDAS_GLOBAL_OCEAN_ANALYSIS_POST [hunk 2]
```diff
@@ -11,9 +11,11 @@ source "${HOMEgfs}/ush/jjob_header.sh" -e "ocnanalpost" -c "base ocnanalpost"
 ##############################################
 
 export COMOUT=${COMOUT:-${ROTDIR}/${CDUMP}.${PDY}/${cyc}/ocean}
```
Comment (guillaumevernieres) [inline jobs/JGDAS_GLOBAL_OCEAN_ANALYSIS_POST:13]: Is there a case in which we would want to use a COMOUT that isn't ```${ROTDIR}/${CDUMP}.${PDY}/${cyc}/ocean```?

Comment (AndrewEichmann-NOAA) [reply jobs/JGDAS_GLOBAL_OCEAN_ANALYSIS_POST:13]: Are we intereacting with other components (eg waves)?

Comment (guillaumevernieres) [reply jobs/JGDAS_GLOBAL_OCEAN_ANALYSIS_POST:13]: no, just ice.

Comment (guillaumevernieres) [reply jobs/JGDAS_GLOBAL_OCEAN_ANALYSIS_POST:13]: ... well, so the answer should have been, yes, we're interacting with ice.

**Pull request merged**